app/models/output_element.rb

```diff
@@ -107,6 +107,18 @@ def has_description?
     description.present?
   end
 
+  # some charts only have one relatee output element, in that case
+  # the hourly/yearly toggle should also show on the current chart
+  def has_one_relatee_output_element?
+    relatee_output_elements.length == 1
+  end
+
+  def relatee_output_element
+    return unless has_one_relatee_output_element?
+
+    relatee_output_elements[0]
```

Comment:
Use `relatee_output_elements.first` if you only want the first object. Like the length/count example, ActiveRecord will be smart enough to only fetch the first output element when using `first`.


```suggestion
    relatee_output_elements.first
```